### ISSUE TITLE
Cannot use bind when it associates to something that requires an argument

### ISSUE THREAD

**Issue by rmpowell77** [rmpowell77/wxUI#86]
I'm finding that I cannot create a bind with a lambda that requires an argument.  `get_arity` doesn't work.

**Comment by rmpowell77** [comment]
Specifically this type of construct fails:
```
                .bind([](auto& e) { (void)e; }),
```

with:
```
[build] In file included from /Users/richardpowell/Development/github.com/rmpowell77/wxUI.git/examples/HelloWorld/ExtendedExample.cpp:27:
[build] In file included from /Users/richardpowell/Development/github.com/rmpowell77/wxUI.git/include/wxUI/wxUI.h:30:
[build] In file included from /Users/richardpowell/Development/github.com/rmpowell77/wxUI.git/include/wxUI/BitmapButton.h:26:
[build] /Users/richardpowell/Development/github.com/rmpowell77/wxUI.git/include/wxUI/Widget.h:50:39: error: reference to overloaded function could not be resolved; did you mean to call it?
[build] struct get_arity : get_arity<decltype(&T::operator())> {
[build]           
...
```

**Comment by rmpowell77** [comment]
Fixed in #87
